publish.sh

```diff
@@ -65,3 +65,4 @@ git add -f \
 printf "OK\n"
 
 printf "\nAll set. **Do not forget to commit new changes.**\n"
+7
```

Comment:
What is going on in this file?

Reply:
Oops, I think my git just converted CRLF to LF line endings. `pub73` stuff - well, when I was trying to understand the build process I made a lot of strange things, I think it's one of them. I thought I've been looking very good to not commit this changes but it appears it wasn't good enough. I'll roll it back.

Reply:
This file doesn't seem to be reverted?

Reply:
@CreepySkeleton  I'd be happy to merge this PR but this file needs to be fixed

Reply:
@Marwes I apologize for the inconvenience. It seems when I merged @ahmedcharles commit a missed some merge artifacts, hence this weird stuff in `publish.sh`.